cdm/core/src/main/java/ucar/nc2/internal/dataset/conv/CF1Convention.java

```diff
@@ -482,13 +482,13 @@ public AxisType getAxisType(VariableDS.Builder vb) {
         return AxisType.Lon;
       }
 
-      if (sname.equalsIgnoreCase(CF.PROJECTION_X_COORDINATE) || sname.equalsIgnoreCase(CF.GRID_LONGITUDE)
-          || sname.equalsIgnoreCase("rotated_longitude")) {
+      if (sname.equalsIgnoreCase(CF.PROJECTION_X_COORDINATE) || sname.equalsIgnoreCase(CF.PROJECTION_X_COORDINATE)
+          || sname.equalsIgnoreCase(CF.PROJECTION_X_ANG_COORDINATE) || sname.equalsIgnoreCase("rotated_longitude")) {
         return AxisType.GeoX;
```

Comment:
Same as above

Reply:
If you look at the source for those two classes, there case of CF.LONGITUDE ->  AxisType.Lon is handled a couple lines up.

Reply:
But what about CF.GRID_LONGITUDE -> AxisType.GeoX? The matching case for CF.GRID_LATITUDE, which is handled here

Reply:
Oh, wow. Looks like I missed your point.

I'm looking at the code changes I made, and it looks like there was a double copy-pasta error.

Will fix in a moment.